src/index.html

```diff
@@ -7,7 +7,81 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <main class="main">
+      <div class="stars stars--0">
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+      </div>
+
+      <div class="stars stars--1">
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+      </div>
+
+      <div class="stars stars--2">
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+      </div>
+
+      <div class="stars stars--3">
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
+
+        <div class="stars__inactive-star"></div>
```

Comment:
the class `inactive` confuses me, because not all of them are actually inactive.
Why don't we just do it like `stars__star`?

Reply:
made the changes, pushing the commit. Many thanks,
Kav.